Rules/ReviewUnusedParameter.cs

```diff
@@ -36,6 +36,12 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
 
             foreach (ScriptBlockAst scriptBlockAst in scriptBlockAsts)
             {
+                // bail out if PS bound parameter used.
+                if (scriptBlockAst.FindAll(IsBoundParametersReference, false).Any())

```

Comment:
Would `Find() != null` be better than `FindAll().Any()`?

Reply:
Yes, it think since it returns the first matching node. Cost effective